src/components/controlButtons/controlButtons.css

```diff
@@ -0,0 +1,25 @@
+.nokiaControlButtons > div {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 5px;
+}
+
+.nokiaControlButtons > div > button:first-child {
+  border: none;
+  padding: 15px 50px;
```

Comment:
maybe transfer these repeating values to the parent element.

Reply:
It might be possible to put the repeating lines into an extra class

Reply:
you are absolutely right!